packages/@ama-sdk/schematics/schematics/helpers/tree-glob.ts

```diff
@@ -0,0 +1,42 @@
+import { DirEntry, Tree } from '@angular-devkit/schematics';
+import * as minimatch from 'minimatch';
+
+const walkThroughDir = (
+  tree: Tree,
+  dir: DirEntry,
+  filterFunction: (
+    element: string,
+    indexed: number,
+    array: readonly string[]
+  ) => boolean,
+  parents = ''
+): string[] => {
+  const folders = dir.subdirs.map((d) => `${parents}/${d}`);
+  const matchingFolders = folders.filter(filterFunction);
+  return [
+    ...dir.subfiles.map((d) => `${parents}/${d}`).filter(filterFunction),
```

Comment:
Nitpicking for readability
```suggestion
    ...dir.subfiles.map((f => `${parents}/${f}`).filter(filterFunction),
```